Core/Utilities/InstallTools.py

```diff
@@ -2,6 +2,7 @@
 # File :    InstallTools.py
 # Author :  Ricardo Graciani
 ########################################################################
+from _mysql import result
```

Comment:
What's the need for this? If there's a need, official documentation is suggesting NOT TO use it:

```python
In [2]: _mysql?
Type:       module
String Form:<module '_mysql' from '/usr/lib/python2.7/dist-packages/_mysql.so'>
File:       /usr/lib/python2.7/dist-packages/_mysql.so
Docstring:
an adaptation of the MySQL C API (mostly)

You probably are better off using MySQLdb instead of using this
module directly.
...
```

and also:

```python
In [3]: from _mysql import result

In [4]: result?
Type:       type
String Form:<type '_mysql.result'>
File:       /usr/lib/python2.7/dist-packages/_mysql.so
Docstring:
result(connection, use=0, converter={}) -- Result set from a query.

Creating instances of this class directly is an excellent way to
shoot yourself in the foot. If using _mysql.connection directly,
use connection.store_result() or connection.use_result() instead.
If using MySQLdb.Connection, this is done by the cursor class.
Just forget you ever saw this. Forget... FOR-GET...
```